tutormfe/patches/local-docker-compose-prod-services

```diff
@@ -4,3 +4,5 @@ mfe:
     build:
         context: ../plugins/mfe/build/mfe/
     restart: unless-stopped
+    depends_on:
+        - lms
```

Comment:
The change makes total sense, but I'm curious: did you actually run into a failure without it?  It seemed to worked on my local tests, but maybe I wasn't fast enough. :)

Reply:
No I didn't, but I would have if I had run `tutor dev start account`... Which I probably already did one day. MFEs depended on the LMS even before dynamic config, so this is not a new failure case.